test/models/user_test.rb

```diff
@@ -0,0 +1,41 @@
+require "test_helper"
+
+require 'pry'
+
+describe User do
+
+  before do
+    @user = User.create!(username: "Julia")
```

Comment:
Why make a user in the tests and why not start with a valid user defined in the fixtures?